### DISCUSSION TITLE
Top bar icon colours.

### DISCUSSION THREAD

**Discussion by Alex313031** [Alex313031/thorium#50] [Ideas]
Thorium has some modified .icon files and .svg files, and I'm starting a conversation about what the coloring should be.

If you want to experiment with the .icon files, you have multiple options.
I created an extension for the chromium repo that will also work with local files > https://chrome.google.com/webstore/detail/chromium-vector-icon-pain/cnlndebkegcfnbiknohgneobakigplhf

And there are two pages on thorium's website, one for viewing/modifying .icon files in realtime > https://thorium.rocks/misc/vector-icon-app/
and another for converting an .svg file to .icon format > https://thorium.rocks/misc/skiafier/skiafy

The .icon files are located in [//chrome/app/vector_icons](https://github.com/Alex313031/Thorium/tree/main/src/chrome/app/vector_icons) and [//components/vector_icons](https://github.com/Alex313031/Thorium/tree/main/src/components/vector_icons)

The colors can be modified by using html color codes.
For example this line in the thorium flags icon https://github.com/Alex313031/Thorium/blob/main/src/chrome/app/vector_icons/chrome_labs.icon#L6 

Ignore the first 2 digits in the code, this sets alpha/transparency and is always FF for no transparency. The other ones after are what you want to modify. For example this icon is using the greenish color #87FFC5. To make it pure black, for example, one could modify that line to `PATH_COLOR_ARGB, 0xFF, 0x00, 0x00, 0x00,`, equivalent to #000000.

You can learn more about the special .icon format Chromium uses for native UI icons [here](https://chromium.googlesource.com/chromium/src/+/main/components/vector_icons/README.md)

I welcome any color changes, or full .icon style changes. You could send me a full .icon file, or just post the html color code or a generic color that you think would look good. See this [issue](https://github.com/Alex313031/Thorium/issues/48#issuecomment-1189297365) that prompted me to make this.

@raisedintheusa

@midzer 

@gz83

**Comment by raisedintheusa** [comment]
I complained about the hamburger menu icon because to me it looks like four blurry overlapping dots make up the bars, but now that I look at the actual .icon file, I see they are in fact solid lines. I'm not sure why it appears this way to me, my mind must want to see dots there for some reason. Or it's my computer screen. The hamburger menu icon in Brave does look like lines to me. Curious why the Thorium one has this effect for me.

As far as an idea, would making a "T" shape for the hamburger icon be too corny? For instance with dots, very roughly,
PATH_COLOR_ARGB, 0xFF, 0x1A, 0x73, 0xE8,
CIRCLE, 40, 10, 4,
CIRCLE, 8, 10, 4,
CIRCLE, 24, 10, 4,
CIRCLE, 24, 25, 4,
CIRCLE, 24, 40, 4,
CLOSE
However, I wouldn't want it too reminiscent of T-Mobile's logo, which also features dots.

**Reply by Alex313031** [reply]
haha I like the idea. I might make a personal build with this to see how it looks.
